mgmtsystem/rest_client.py

```diff
@@ -2,6 +2,7 @@
 import os
 import json
 from exceptions import RestClientException
+from utils import eval_strings
```

Comment:
please either use explicit relative imports or absolute imports

implicit relative imports are problematic and not forward compatible to python3